tasks/build/Gaia/COMMIT.sh

```diff
@@ -2,4 +2,4 @@
 
 # This is the commit of the SDK version to use for building Gaia.  We use an
 # explicit hash instead of a tag so we don't have to trust GitHub.
```

Comment:
```suggestion
```

Reply:
.... why are we switching to tags if this means trusting github?

Reply:
Because if the SDK team decides to switch the version tag to another commit, we are covered. it is easier to identify and talk about a version tag then about a cryptic commit hash.

Reply:
good reasoning.

Reply:
Plus we have the tagged docker images :) as soon as we have time we should remove all the building here and just run that container